src/Skyroom/Menu/UserSubmenu.php

```diff
@@ -49,7 +43,7 @@ public function __construct(UserRepository $repository, Viewer $viewer)
     function display()
     {
         try {
-            $users = $this->repository->getUsers();
+            $users = $this->woocommerceAdaptor->getSkyroomUsers();
```

Comment:
Will be better to remove this menu completely as we don't have anything to do with skyroom users.

Reply:
No, This is a good feature for having a list of users

Reply:
As we have nothing to do with skyroom users in wordpress plugin, having this menu may confuse users while it doesn't add any value for them.

Reply:
Incidentally, since nothing is registered on our site, being in this section helps the customer a lot